roles/zuul/files/jobs/hoist.yaml

```diff
@@ -22,3 +23,4 @@
       - shell: |
           ./tests/signed-off-by-test.sh
           python tests/layout-checks.py job-list.txt
+          ./tests/shellcheck-test.sh
```

Comment:
I'm not sure why that would be needed since it seems to take each line as a separate thing to run and it goes off and runs each one (all starting at the same time).